lib/google/ads/google_ads/logging_interceptor.rb

```diff
@@ -52,7 +52,7 @@ def request_response(request:, call:, method:, metadata: {})
             response_message = "Incoming response (errors): \n"
 
             most_recent_error = Google::Gax::GaxError.new('')
-            most_recent_error.status_details.each do |detail|
+            most_recent_error.status_details&.each do |detail|
```

Comment:
The &. operator was added in Ruby 2.3. We'd prefer not to restrict support to earlier versions unnecessarily, even if they are rarely used. Can you implement this a different way?